src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,16 @@
+package duke.task;
+
+public class Todo extends Task {
+
+    public Todo(String description, boolean done) { super(description, done); }
```

Comment:
I am not so sure about having a single-lined method declaration and code, might need to check with others/prof/TAs about it.  
Referring to the Google Java Style Guide (for stuff not on the module site), we need to break it up to:
``` java
public Todo(String description, boolean done) {
    super(description, done);
}
```
See: https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style

![image](https://user-images.githubusercontent.com/66059643/131637726-dd888b41-d8eb-4cc2-a680-91f2867e8afc.png)